app/models/article.rb

```diff
@@ -129,6 +112,22 @@ def lede
 
   private
 
+  def find_related_articles
+    return {} if categories.blank?
+
+    # we do not want the current article to show up in related articles
+    articles_to_exclude = [id]
+    categories.each_with_object({}) do |category, hash|
+      # get 3 articles that we haven't seen yet in a previous iteration of this loop
+      related_articles = category.articles.english.published.limit(3).where.not(id: articles_to_exclude)
```

Comment:
In a lot of place in the codebase we use `live.published` is there a reason why here we only use `published`?

Reply:
no, idea. I just copied the existing code. I think you are correct, let's throw `live` in there 

```suggestion
      related_articles = category.articles.english.live.published.limit(3).where.not(id: articles_to_exclude)
```